src/graph_compute/common.h

```diff
@@ -35,6 +35,10 @@
   #define WHEN_DEBUG(ex)
 #endif
 
+#ifndef NUM_CORES
+  #define NUM_CORES 12
```

Comment:
What's the benefit of having a separate switch for `NUM_CORES` vs just using `NUMA_WORKERS`? This is a bit misleading because `NUM_CORES` can't be used with Cilk as far as I can tell, and I don't like it.